.github/workflows/build_frontends.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Setup Biome
         uses: biomejs/setup-biome@v2
         with:
-          version: latest
+          version: 1.8.3
```

Comment:
It would be better if we'd use the version that we've specified in the package.json